certified-connectors/PartnerLinq/apiDefinition.swagger.json

```diff
@@ -0,0 +1,190 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "PartnerLinq",
+    "description": "PartnerLinq is an integration tool used to connect two or more systems together. Use this connector to connect with PartnerLinq.",
+    "version": "1.0",
+	"contact":{
+		"name":"PartnerLinq Support",
+		"url":"https://www.partnerlinq.com/",
+		"email":"PL.Supportteam@visionet.com"
+	}
+  },
+  "host": "pod-dev1-cus-func-httpdatareceive.azurewebsites.net",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/api/FUNC_HTTP_DATA_SEND": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
```

Comment:
@FarhanJamilVSI empty schema is not allowed. Please remove or define a proper schema.
Please fix all empty schemas in your connector swagger file.